README.md

```diff
@@ -1658,6 +1659,47 @@ aem {
 
 There are also available convenient methods `asStream`, `asString` to be able to process endpoint responses.
 
+#### Making changes to repository
+
+To make some changes in repository on AEM instance use `nodes` namespace in `aem.sync`
```

Comment:
if you will review other examples, you could spot that they are trying to cover real life scenarios.

and I'd like to encourage you to invent sth. people get involved when they see real examples / mocks are not so demoable :)